src/Paket/Program.fs

```diff
@@ -49,6 +49,9 @@ type CLIArguments =
     | [<AltCommandLine("-f")>] Force
     | Hard
     | [<CustomCommandLine("nuget")>] Nuget of string
+//    | [<CustomCommandLine("gist")>] Gist of string * string
+//    | [<CustomCommandLine("github")>] Github of string * string
+    | [<CustomCommandLine("url")>] Url of string * string
     | [<CustomCommandLine("version")>] Version of string
     | [<CustomCommandLine("add-credentials")>] AddCredentials of string
```

Comment:
Eirik, I've created the said issue.

Forki, I am not sure if using more than one `[<Rest>]` is possible. Maybe I don't fully understand that attribute, but I don't see how using 2 of them would make sense.
I'll take a look at what's being done in #551 tomorrow.